cicecore/cicedynB/analysis/ice_history.F90

```diff
@@ -1814,7 +1815,7 @@ subroutine accum_hist (dt)
       n4Dfcum = n4Dscum + num_avail_hist_fields_4Df ! should equal num_avail_hist_fields_tot
 
       do ns = 1,nstreams
-         if (.not. hist_avg .or. histfreq(ns) == '1') then  ! write snapshots
+         if (.not. hist_avg) then  ! write snapshots
```

Comment:
At some point, I would like hist_avg to be an array of size nstreams. Then we can control the instantaneous versus average on a stream by stream basis.

Reply:
That's a good idea, I will make sure that's part of an issue.